djangocms_pageadmin/test_utils/factories.py

```diff
@@ -112,3 +112,13 @@ def version(self, create, extracted, **kwargs):
             # Simple build, do nothing.
             return
         PageVersionFactory(content=self, **kwargs)
+
+
+class PageUrlFactory(factory.django.DjangoModelFactory):
+    slug = ''
+    path = ''
+    managed = False
+    language = 'en'
```

Comment:
```suggestion
    language = "en"
```
The rest of this file has double quotes for strings so better to keep it consistent. Have any tools like black been considered to keep consistency like this? Or just a flake8 rule perhaps.

Reply:
Yep, will look into adding that, it's definitely something I've noticed all of us missing in the past, so certainly worth the time.